README.md

```diff
@@ -113,26 +113,14 @@ There is a GitHub action workflow that automatically runs this against PRs to en
 
 The standard geographies used in this tool are sourced from (or developed with) the Office for National Statistics (ONS). As such, they are maintained in line with the ONS's [open geography portal](https://geoportal.statistics.gov.uk/). 
 
-#### Updating pcon_la or lad_lsip lookups
+#### Updating ward_pcon_lad_la or lad_lsip lookups
 
-There's an R script in the R/standard-data-prep folder containing functions to update both the pcon_la and lad_lsip lookups. To use this, download the latest version of the file from the searches linked in the comments of that script, and then save a copy of that data in a new folder called /data/downloaded_source_data/. The functions will then append any new data from that CSV into the lookup files.
+There's an R script in the `R/manual_scripts/` folder containing functions to update both the geography lookups. To use this, download the latest version of the file from the searches linked in the comments of that script, and then save a copy of that data in a new folder called `/data/downloaded_source_data/`. The functions will then append any new data from that CSV into the lookup files.
```

Comment:
Can we add a specific link for a search that returns the lookups on open geography? If you don't know your way around the portal then it can be hard finding what you need.

Reply:
The links are in the script itself for the specific functions. Is there anything that would make that clearer here?

Reply:
Going to merge this so the data is in for guide to the galaxy, but can agree the text updates afterwards